apis/frontend/js-apis/record-attribution.md

```diff
@@ -1,12 +1,11 @@
----
-description: >-
-  The `recordAttribution` API combines the functionality of triggering `vp`
-  event and a `src` event.
----
-
 # Record Attribution
 
-Input parameters to this API is similar to the Session API version with one minor difference. JS API version accepts an \`Event\` object as a parameter whereas the former version accepts them as individual parameters
+The `recordAttribution` method provides a low-level standalone API to submit events to Nosto for processing in the backend.
+It is to be used for standalone attribution submission that are not covered by the alternatives:
+* Parameterless attribution
+* Usage of attribution url parameters
+* Session API
+* Request API
```

Comment:
Is the Request API naming already exposed to the merchants? Otherwise it would confuse them

Reply:
You are right, not used
Shall I use add this as explanation: api.createRecommendationRequest()

Reply:
updated